src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const littleDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days <= 2) {
+    return days * price;
```

Comment:
`days * price` - you use this expression in several places, so makes sense to move it to a variable